lib/screen/customer_support/support_list_page.dart

```diff
@@ -62,11 +64,17 @@ class _SupportListPageState extends State<SupportListPage>
         .removeListener(loadIssues);
   }
 
+  Future<List<ChatThread>> _loadChatThreads() async {
+    final chatThreads =
```

Comment:
should add try...catch for async function

Reply:
I dont think we should add try cache here as it's sync function, that we called when init state. If any error, it will be handled in main.